Cargo.toml

```diff
@@ -159,6 +159,7 @@ sqlx = "0.7.3"
 static_assertions = "1.1"
 structopt = "0.3.20"
 strum = "0.24"
+strum_macros = "0.24"
```

Comment:
I don't think importing this crate directly is necessary; other `strum` uses in the workspace import it via

```toml
strum = { workspace = true, features = ["derive"] }
```